app/src/main/res/drawable/btn_keyboard_key_lxx_dark_amoled.xml

```diff
@@ -3,9 +3,9 @@
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <!-- Action keys. -->
     <item android:state_active="true" android:state_pressed="true"
-          android:drawable="@color/key_background_pressed_lxx_dark" />
+        android:drawable="@drawable/btn_keyboard_key_action_pressed_lxx_dark" />
     <item android:state_active="true"
-          android:drawable="@color/background_amoled_black" />
+        android:drawable="@drawable/btn_keyboard_key_action_normal_lxx_dark" />
```

Comment:
This is the only change that doesn't (directly) have to do with Android S theming support. I thought the AMOLED keyboard looked a bit better with some color behind the "Enter" key, so I updated it to use the same colors as the material dark keyboard (and it will follow the system accent, per the rest of this PR).